### ISSUE TITLE
Problem about accuracy

### ISSUE THREAD

**Issue by ujscjj** [thuml/MDD#2]
Hello, Thank you for your reply about accuracy.
Yes, it is a problem about pytorch version.
In pytorch1.0, I just change the code:
accuracy = torch.sum(torch.squeeze(predict).float() == all_labels) / float(all_labels.size()[0]) in train.py to --->
accuracy = Tensor.cpu(torch.sum(torch.squeeze(predict).float() == all_labels)).numpy() / float(all_labels.size()[0])

It will run rightly.
Thank you for your reply again.

**Comment by hlqwerty** [comment]
Thank you very much for helping us find the error! We will also rewrite our code with a new pytorch version in the future.
